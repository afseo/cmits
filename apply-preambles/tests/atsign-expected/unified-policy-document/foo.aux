contents of a aux file
