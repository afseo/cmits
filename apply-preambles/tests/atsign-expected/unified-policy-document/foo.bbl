contents of a bbl file
