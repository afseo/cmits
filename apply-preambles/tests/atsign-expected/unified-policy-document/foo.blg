contents of a blg file
