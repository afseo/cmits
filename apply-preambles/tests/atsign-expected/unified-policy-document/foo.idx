contents of a idx file
