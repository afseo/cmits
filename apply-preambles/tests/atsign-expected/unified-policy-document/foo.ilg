contents of a ilg file
