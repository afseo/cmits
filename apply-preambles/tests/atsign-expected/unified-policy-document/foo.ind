contents of a ind file
