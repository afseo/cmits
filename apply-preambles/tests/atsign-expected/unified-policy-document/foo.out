contents of a out file
